src/type_resolve_helpers.ts

```diff
@@ -144,7 +145,49 @@ function generateTree(name: string, parent: StringTreeNode | null = null) : Stri
             continue;
         }
 
-        // TODO: Function?
+        // Function
+        if (partUpper === 'FUNCTION')
```

Comment:
Not something that needs to be done here, but I wonder if we can use the Typescript library to parse these function strings rather than doing it ourselves.

Reply:
Agreed